.env.dist

```diff
@@ -39,6 +42,8 @@ MAIL_TO_ERROR_REPORT_ADDRESS=example@example.com
 MAIL_TO_ERROR_REPORT_NAME='Database Wizards'
 MAIL_TO_SUBSCRIPTION_ADDRESS=example@example.com
 MAIL_TO_SUBSCRIPTION_NAME='Secretary of GEWIS'
+MAIL_TO_PUBLIC_ADDRESS=example@example.com
+MAIL_TO_PUBLIC_NAME='Secretary of GEWIS'
```

Comment:
I recommend changing this to `MAIL_FROM_SECRETARY_ADDRESS` and `MAIL_FROM_SECRETARY_NAME`, as we are not sending the e-mails to the secretary but effectively replacing the `From`.

Reply:
Done, https://github.com/GEWIS/gewisdb/pull/286/commits/76f4efbe7a8985a19a98271a70db39ac3aef9c75